bin/templates/scripts/cordova/build

```diff
@@ -42,6 +42,7 @@ var buildOpts = nopt({
     'codeSignIdentity': String,
     'codeSignResourceRules': String,
     'provisioningProfile': String,
+    'multipleProvisioningProfiles': Array,
```

Comment:
I wonder if we should add a new property here for `multipleProvisioningProfiles` vs changing the existing `provisioningProfile` property to accept either a single string or an array.

Reply:
I guess thats a better approach - i will change that.

Reply:
I have to revert the idea - merging the two options do not work and require more refactoring. So I suggest to add this additional parameter which is only used for the export plist.